addons/crm/static/src/js/tours/crm.js

```diff
@@ -1,100 +1,113 @@
+odoo.define('crm.crm_steps', function (require) {
+"use strict";
+
+var core = require('web.core');
+
+var CrmAdditionalTourSteps = core.Class.extend({
+
+    _get_crm_iap_lead_steps: function () {
+        return [];
+    },
+
+});
+
+return CrmAdditionalTourSteps;
+
+});
+
 odoo.define('crm.tour', function(require) {
 "use strict";
 
 var core = require('web.core');
 var tour = require('web_tour.tour');
+var CrmAdditionalTourSteps = require('crm.crm_steps');
 
 var _t = core._t;
 
 tour.register('crm_tour', {
     url: "/web",
+    rainbowManMessage: _t("Congrats, best of luck catching such big fish! :)"),
 }, [tour.stepUtils.showAppsMenuItem(), {
     trigger: '.o_app[data-menu-xmlid="crm.crm_menu_root"]',
-    content: _t("Ready to boost your sales? Your <b>Pipeline</b> can be found here, under the <b>CRM</b> app."),
-    position: 'right',
-    edition: 'community'
+    content: _t('Ready to boost your sales? Let\'s have a look at your <b>Pipeline</b>.'),
+    position: 'bottom',
+    edition: 'community',
 }, {
     trigger: '.o_app[data-menu-xmlid="crm.crm_menu_root"]',
-    content: _t("Ready to boost your sales? Your <b>Pipeline</b> can be found here, under the <b>CRM</b> app."),
+    content: _t('Ready to boost your sales? Let\'s have a look at your <b>Pipeline</b>.'),
     position: 'bottom',
     edition: 'enterprise',
 }, {
-    trigger: ".o-kanban-button-new",
+    trigger: '.o-kanban-button-new',
     extra_trigger: '.o_opportunity_kanban',
-    content: _t("Click here to <b>create your first opportunity</b> and add it to your pipeline."),
-    position: "bottom",
+    content: _t("<b>Create your first opportunity.</b>"),
+    position: 'bottom',
 }, {
-    trigger: ".o_kanban_quick_create input:first",
-    content: _t("<b>Choose a name</b> for your opportunity, example: <i>'Need a new website'</i>"),
+    trigger: ".o_kanban_quick_create .o_field_widget[name='partner_id']",
+    content: _t('Look for a Contact or create a new one.<br><i>Tip : Did you know you can search by VAT Number as well?</i>'),
     position: "right",
+    run: function (actions) {
+        actions.text("Brandon Freeman", this.$anchor.find("input"));
```

Comment:
While I was debugging your issue, I noticed that this did nothing.

Can you double check?

Reply:
when running the tests it fills the input "partner_id" with this name (Brandon Freeman).

Reply:
When I ran the tour on my end it did not work, I had to input the name myself.

But if you tell me it works now I trust you.